src/ui/chip/Chip.tsx

```diff
@@ -13,7 +13,7 @@ export const Chip = ({ label }: ChipProps) => {
 
   return (
     <MUIChip
-      label={formatMessage({ id: `rankingTable.${label}` })}
+      label={description ? description : formatMessage({ id: `rankingTable.${label}` })}
```

Comment:
Czyli jak używamy `description` to i tak trzeba podać `label` mimo że nie jest używany?
I dopiero teraz zauważyłem - zauważ, że aktualnie `Chip` jest powiązany z `rankingTable`. Czyli jest średnio reużywalny, powinien się wręcz nazywać `RankingChip`. Dobrze by było, żeby był tylko podawany `label` jako string który będzie tam bezpośrednio wyświetlany